client/components/Help/Help.js

```diff
@@ -2,37 +2,75 @@ import React, { Component, PropTypes } from 'react';
 import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
 import { connect } from 'react-redux';
 import { NewProblem } from '../NewProblem/NewProblem';
-import { Problem } from './../Problem/Problem';
-import { getProblems, addProblem } from '../../actions/actions';
+import { Problem } from '../Problem/Problem';
+import { AnswersList } from '../AnswersList/AnswersList';
+import { getProblems, addProblem, showCurrentProblem } from '../../actions/actions';
 import './Help.scss';
 
 class Help extends Component {
     state = {
-        isNewProblemVisible: false
+        isNewProblemVisible: false,
+        isAnswersListVisible: false
     };
 
-    componentDidMount() {
-        this.props.dispatch(getProblems());
-    }
-
     newButtonClick = () => {
-        this.setState({
-            isNewProblemVisible: true
-        });
+        if (this.state.isAnswersListVisible) {
+            this.setState({
+                isAnswersListVisible: false
+            });
+            setTimeout(() => {
+                this.setState({
+                    isNewProblemVisible: true
+                });
+            }, 500);
+        } else {
+            this.setState({
+                isNewProblemVisible: true
+            });
+        }
     };
 
     addButtonClick = (problem) => {
-        this.props.dispatch(addProblem(problem));
+        this.props.addProblem(problem);
         this.setState({
             isNewProblemVisible: false
         });
     };
 
+    viewButtonClick = (currentProblem) => {
+        this.props.showCurrentProblem(currentProblem);
```

Comment:
is it necessary to save selected problem to redux store?? just save it to state, it is local data of your view